src/python/pants/backend/codegen/protobuf/target_types.py

```diff
@@ -17,13 +17,7 @@
 
 
 class ProtobufDependencies(Dependencies):
-    """Addresses to other targets that this target depends on, e.g. `['protobuf/example:lib']`.
-
-    Pants will automatically inject any targets that you configure in the option `runtime_targets`
-    in the `[protoc]` scope. For example, if you set that option to include the Python runtime
-    library for Protobuf, every `protobuf_library` will automatically include that in its
-    `dependencies`.
```

Comment:
This option is deprecated. Its replacement is in `[python-protobuf]`, so it's wrong to talk about it in the generic `Dependencies` field for a `protobuf_library`. In some feature, you might activate `pants.backend.codegen.protobuf.java`, and that `[python-protobuf]` option would never be used.